python-sdk/src/astro/databricks/databricks_api_utils.py

```diff
@@ -0,0 +1,144 @@
+import fileinput
+import pathlib
+import time
+from pathlib import Path
+from typing import Dict
+
+from attrs import define, field
+from databricks_cli.dbfs.api import DbfsApi, DbfsPath
+from databricks_cli.jobs.api import JobsApi
+from databricks_cli.runs.api import RunsApi
+from databricks_cli.sdk.api_client import ApiClient
+
+from astro.databricks.load_file.load_file_python_code_generator import render
+
+cwd = pathlib.Path(__file__).parent
+
+
+@define
+class DeltaOption:
+    """
+    Helper class for rendering options into COPY_INTO Databricks SQL statements
+    """
+
+    option_dict: dict = field(factory=dict)
+
+    def empty(self):
+        return len(self.option_dict or {}) == 0
+
+    def convert_to_option_string(self):
+        return ",".join([f"\\'{k}\\' = \\'{v}\\'" for k, v in self.option_dict.items()])
```

Comment:
Why do we need to escape?

```python
In [6]:     def convert_to_option_string(self):
   ...:         return ",".join([f"\\'{k}\\' = \\'{v}\\'" for k, v in self.option_dict.items()])
   ...:

In [7]:     def convert_to_option_string(self):
   ...:         return ",".join([f"'{k}' = '{v}'" for k, v in self.option_dict.items()])
   ...:
```

Reply:
Ah yes sounds good. This is a leftover of trying to get around some jinja weirdness when it comes to internal quotes. I should be able to fix this.